README.md

```diff
@@ -1 +1,47 @@
+
+
+
+## Problems :
+* The main problem that makes us cearte this wibsite is the daialy challeng for pepole " What to eat for today?". But still using the ingredients in the fridge.
+* The other problem is people / users may wonder what the restaurant in some area and need information about restaurants. 
+ 
+## The Soluation :
+* Our website provide the ability to access all the receipts that might know our not in how to make it with the ingredients that have already exists in your fridge. Also provide a link of YouTube video to make it easy to user to follow the chief do the choosing receipt.
+* The solution for the other problem is to make section for finding restaurants depending on specific area and provide all information your need to know about each restaurant.
+
+## wireframes :
+
+
+## User journey :
+First of all, our site starts with “splash page” that make user enter the " Bonappetit " by clicking on “get started button” 
+The site divided into 4 pages (Home, inspiration, Restaurants, Contact us)
+Home page will contain 
+Navigation bar to make the user move between pages easily, and the section of Recipes generator that the user can write any ingredient by typing it in the search box and will get all recipes use this ingredient and when clicking on one of the recipes will appear a pop-up window contain the ingredients of this recipes and a link of YouTube video to how cook this recipe.
+The next page is Inspiration page by click inspiration link on navigation bar, this page contains a button called “Find your recipe for today”, this button may help you choose your recipe if you want to try something new, just don’t know what to eat today and get out of your comfort zone, so will appear a pop-up window that contain the random recipe and all the ingredient that you need also a you tube link to follow the chef doing this chosen recipe.
+If the user wants to know the restaurants in a specific location/area we create a page for this. the user can enter this page by clicking on the link that is on navigation bar and he can find a search box and write whatever the location he wants the restaurants from, then any restaurant he click on will appear a pop-up window contain the menu of chosen one and a specific location to make it easy to reach it out, also the link of the restaurant website.
+Contact page is something necessary of our page if you have any improvement suggestion or any problem , so feel free to contact us.
+
+
+## User story :
+* As a user who loves to prepare food so I need a website help me to find all recipes from the ingredients that I have in my fridge to make it easy to me decide what I want meal for today.
+* also, I need the website decide me sometimes and send me a random meal to try something new and different of what I eat as usual. 
+* it would be nice if the website have a section to get me the restaurants in some location and information about this restaurant in case I want to eat outside my house.
```

Comment:
**User stories should explain the services that a user can use as a list, 
So, Fix the syntax of it as the following :** 

* As a user I can, find all recipes from the ingredients that I have in my fridge to make it easy to me decide what I want meal for today.

// fix the rest of them.